src/test/java/org/broadinstitute/hellbender/tools/picard/sam/CreateSequenceDictionaryIntegrationTest.java

```diff
@@ -46,4 +46,17 @@ public void testNonUniqueSequenceName() throws Exception {
         runCommandLine(argv);
         Assert.fail("Exception should have been thrown.");
     }
+
+    // Should throw an exception because no reference file was specified
+    @Test(expectedExceptions = {UserException.CommandLineException.class})
+    public void testCustomCommandLineValidation() throws Exception {
+        final File output = File.createTempFile("TestOutput",".out");
+        output.deleteOnExit();
+        final String[] argv = {
+                "CreateSequenceDictionary",
+                "--OUTPUT", output.getAbsolutePath()
+        };
+        runCommandLine(argv);
+        Assert.fail("Exception should have been thrown.");
```

Comment:
No need for this assert -- TestNG's `expectedExceptions` will handle this for you automatically